frontend/src/views/MainMenu.vue

```diff
@@ -11,6 +11,15 @@
                         <img class="bulb" src="../assets/images/bulb.png" />
                     </div>
                 </div>
+              
+                <div id="app">
+                  <button @click()="playM">play music</button>
+
+                </div>
+
+                <div v-if="musicPlays">
```

Comment:
I think maybe this would be better to have as a toggle button, there's a good example with the mute chat button on the game page. 

When music is playing we could make the button red or change logo or something similar.

Reply:
yeah this sounds better since i was also thinking about a mute feature for the music